test/lib/stack-client/stack-client.init.spec.js

```diff
@@ -31,44 +44,29 @@ describe('stack client', () => {
     })
 
     it('should not called internal client', async () => {
-      await init(params)
+      isLogged = false
+      await setup()
```

Comment:
Maybe we should pass `isLogged/isPublic` as argument of the `setup` function with an options object like `setup({ isLogged: false })`?

Reply:
Same same ? I see no direct advantage of doing this? No strong opinion, I keep that in mind though.

Reply:
The main advantage is to not rely on a global variable and no need to be sure to reset it each time. It's a small advantage I know